app/assets/stylesheets/_sponsor_application.scss

```diff
@@ -1,80 +1,70 @@
-*,
-*:after,
-*:before {
-  box-sizing: border-box;
+.status {
+  margin: 20px 0;
+  clear: both;
 }
 
-.steps-container {
-  overflow: hidden;
-  margin: 0px;
-  padding: 0px;
-  white-space: nowrap;
-  border-left: 2px solid;
-  border-right: 2px solid;
-  width: 100%;
-  counter-reset: steps;
+.status__heading {
+  line-height: 25px;
```

Comment:
Properties should be ordered float, font-weight, line-height, margin, margin-bottom, margin-left